lib/core/src/lib/auth/basic-auth/basic-alfresco-auth.service.ts

```diff
@@ -351,19 +351,18 @@ export class BasicAlfrescoAuthService extends BaseAuthenticationService {
      * @returns The ticket or `null` if none was found
      */
     getTicketEcmBase64(requestUrl: string): string | null {
```

Comment:
I think null type as return is redundant now and it's enough to leave string type here. Earlier there was 
let ticket = null; 
and 
return ticket; 

so null type had sense but now there can't be null I think